src/lib/workflow.ts

```diff
@@ -22,6 +22,8 @@ export interface OnPaths {
   "paths-ignore"?: string[];
 }
 
+export type WorkflowDispatchInputsType = 'boolean' | 'number' | 'string' | 'choice';
```

Comment:
Couple things:
- there is no support for `number` for `workflow_dispatch`
- there _is_ support for an `environment` input type

Reply:
Fixed. I got confused over `workflow_call` which has a `number` option. 😅